govcd/catalog.go

```diff
@@ -503,6 +503,10 @@ func queryVappTemplateAndVerifyTask(client *Client, vappTemplateUrl *url.URL, ne
 		return nil, err
 	}
 
+	if vappTemplateParsed.Tasks == nil {
```

Comment:
The only thought I have here is - can we do something more in this case of error? Another refresh? Some more information to put into logs (if we need to retrieve anything as otherwise, that would already be in the logs) to be able to capture the situation.

Reply:
I forgot your comment, sorry. I've added a log entry that dumps the whole vappTemplateParsed structure. Not sure if it will be useful, but it's more info than before.

Reply:
BTW, to overcome this issue, what I had to do in UI is delete the media item (it gets uploaded as media instead of vApp Template) and re-upload again. So a proper fix wouldn't be trivial in this case and would require a separated PR.

Reply:
> I've added a log entry that dumps the whole vappTemplateParsed structure.
IMO this is good enough, thanks. It sounds like the case is related to situations when upload fails